test/studies/shootout/fasta/bradc/fasta-blc.chpl

```diff
@@ -0,0 +1,152 @@
+/* The Computer Language Benchmarks Game
+   http://benchmarksgame.alioth.debian.org/
+
+   contributed by Casey, Battaglino, Kyle Brady, Preston Sahabu, 
+               and Brad Chamberlain
+*/
+use IO;
+
+config const n = 1000;   // controls the length of the generated strings
+
+config const lineLength = 60,
+             lookupSize = 4*1024,
+             lookupScale = lookupSize - 1.0;
+
+//
+// TODO: Could we eliminate these ascii initializers somehow?
```

Comment:
The idea I had was to use the default enums and then do the enum:string:int(8) conversion at the binary I/O phase.  My guess is that the runtime overhead to do additional conversions in the I/O step would be too great and that we should just pay the style cost of doing these initializations here.  But I hadn't tried it yet which is why I was wondering.

Alternatively, we could potentially move away from binary I/O and use text I/O in which case the I/O on enums would print out the characters.  Here too, my guess is that the I/O costs would grow beyond what we'd want, but again, it seems reasonable to try.